tsl/src/remote/connection.c

```diff
@@ -1150,6 +1168,9 @@ remote_connection_open_with_options_nothrow(const char *node_name, List *connect
 	int option_count;
 	int option_pos;
 
+	if (NULL != errmsg)
+		*errmsg = NULL;
```

Comment:
This seems to be unnecessary and instead should be initialised correctly by the caller.